paddle/fluid/operators/fill_zeros_like_op.cu.cc

```diff
@@ -22,4 +23,6 @@ REGISTER_OP_CUDA_KERNEL(
     ops::FillZerosLikeKernel<paddle::platform::CUDADeviceContext, int64_t>,
     ops::FillZerosLikeKernel<paddle::platform::CUDADeviceContext, float>,
     ops::FillZerosLikeKernel<paddle::platform::CUDADeviceContext, double>,
+    ops::FillZerosLikeKernel<paddle::platform::CUDADeviceContext,
```

Comment:
You had better add `InferVarType` for `fill_zeros_like `, otherwise the dtype of the output var maybe not fp16.